docs/src/app/components/Divider/ExampleForm.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Divider from 'material-ui/lib/divider';
+import Paper from 'material-ui/lib/paper';
+import TextField from 'material-ui/lib/text-field';
+
+const DividerExampleForm = () => {
+  return (
+    <Paper zDepth={2}>
+      <TextField hintText="First name" underlineStyle={{display: 'none'}} style={{marginLeft: 20}} />
```

Comment:
True, no reason to be lazy now! That being said, after we merge #2476 , I was planning on coming back to this example to use to use that `underlineShow` prop instead of the `underlineStyle`. Would still need a variable like you said something for the left margin.

If we #2476 is close and we can merge first, I can fix this once and for all.